xformers/csrc/attention/hip_fmha/ck_tiled_fmha_batched_backward.h

```diff
@@ -42,12 +45,18 @@ struct batched_backward_causalmask_bias_dropout_dispatch {
       typename FmhaBwdTypeConfig<ScalarType>::BiasGradDataType,
       FmhaBwdShape<MaxK>,
       false, // kIsGroupMode
+      false, // kIsDeterministic
       FmhaMask,
+      FmhaBlockDropout,
       FmhaTraits>;
 
+  static constexpr bool NeedConvertGradQ = !std::is_same<
+      typename FmhaBwdTypeConfig<ScalarType>::AccDataType,
+      typename FmhaBwdTypeConfig<ScalarType>::QGradDataType>::value;
+
   static void Run(BatchedBackwardParams& param, hipStream_t stream) {
     {
-      constexpr ck_tile::index_t kBlockSize = 256;
+      constexpr ck_tile::index_t kBlockSize = 64;
```

Comment:
Curious about the reason on decreasing block size?

Reply:
Just keep consistent with ck_tile codes 

[fmha_bwd.py #L524](https://github.com/ROCm/composable_kernel/blob/develop/example/ck_tile/01_fmha/codegen/ops/fmha_bwd.py#L524)

which is well tested for performance consideration